node/src/main/scala/coop/rchain/node/storage/RNodeKeyValueStoreManager.scala

```diff
@@ -0,0 +1,61 @@
+package coop.rchain.node.storage
+
+import java.nio.file.Path
+
+import cats.effect.Concurrent
+import cats.effect.concurrent.{Deferred, Ref}
+import cats.syntax.all._
+import coop.rchain.shared.Log
+import coop.rchain.store.{KeyValueStore, KeyValueStoreManager}
+
+object RNodeKeyValueStoreManager {
```

Comment:
Comment here explaining whats the purpose of `RNodeKeyValueStoreManager ` would be helpful.